src/runtime-rs/crates/resource/src/manager.rs

```diff
@@ -98,6 +98,16 @@ impl ResourceManager {
         inner.dump().await
     }
 
+    pub async fn setup_sandbox_shared_volumes(&self) -> Result<()> {
```

Comment:
I think we should check the hypervisor's capability to support `sharefs` here. like [this](https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/resource/src/manager_inner.rs#L79)

Reply:
OK